dll/appcompat/apphelp/hsdb.c

```diff
@@ -26,11 +26,18 @@ typedef struct _ShimData
     SDBQUERYRESULT Query;
     WCHAR szLayer[MAX_LAYER_LENGTH];
     DWORD unknown;  // 0x14c
+    GUID CompatContext;
```

Comment:
This might actually break a few apitests, I'll investigate.

Reply:
Yeah...
https://reactos.org/testman/diff.php?id1=36738007&id2=36739994&type=1&strip=1

To be continued

Reply:
I can re-use the unknown DWORD to store the version for now, that will not work with a windows apphelp, but the struct size will be the same, and it'll simplify `RosGetProcessCompatVersion`